synapse/rest/media/v1/thumbnail_resource.py

```diff
@@ -85,7 +85,9 @@ def _respond_local_thumbnail(self, request, media_id, width, height,
         media_info = yield self.store.get_local_media(media_id)
 
         if not media_info or media_info["quarantined_by"]:
-            logger.info("Media is quarantined")
+            if media_info:
```

Comment:
fwiw I'd have gone with:

```
if not media_info:
    respond_404(request)
    return
if media_info["quarantined_by"]:
    logger.info("Media is quarantined")
    respond_404(request)
    return
```

but fine

Reply:
This is as well inconsistent to lines [127-128](https://github.com/matrix-org/synapse/pull/2789/files#diff-ef88214e96f37d33dab786fcc1c56c5bR127)

Reply:
I swear to god I just copied and pasted that. Today is not going well.